status/flowcells.py

```diff
@@ -289,4 +289,16 @@ def post(self, flowcell):
                 self.set_header("Content-type", "application/json")
                 self.finish(json.dumps(links))
 
+#Functions
+def get_container_from_id(flowcell):
+    if flowcell[7:].startswith('00000000'):
```

Comment:
hej, yes like @guillermo-carrasco said string `00000000` will always be on `MiSeq` run, so guess this will work. But I am always worried about the number of zeros though, maybe its always the same but I am still paranoid :sweat_smile: . 

So if its **me** I will check for `-` in flowcell name i.e. `if '-' in flowcell[7:]` or `if '-' in flowcell.split('_')[-1]`